extensions/cdc-mysql/src/test/java/com/hazelcast/jet/cdc/mysql/MySqlCdcWhiteBlackListIntegrationTest.java

```diff
@@ -347,13 +347,13 @@ private List<String> allTable0ExpectedOperations(int dbSuffix) {
         int id3 = 1000 * dbSuffix + 3;
         int id4 = 1000 * dbSuffix + 4;
         return Arrays.asList(
-                id1 + "/0:INSERT:TableRow {id=" + id1 + ", value1=" + db + "_table0_val1_0, "
```

Comment:
why do we need the changes in this file? Please rollback, if they are leftover.

Reply:
Force pushed a rebase which should fix this